tests/add-to-cart.spec.ts

```diff
@@ -0,0 +1,18 @@
+import { test, expect } from '@playwright/test';
+
+
+test('test', async ({ page }) => {
+  await page.goto('https://practicesoftwaretesting.com/');
```

Comment:
I'd move the url to the config file https://playwright.dev/docs/test-use-options#basic-options